examples/resources/groundcover_ingestionkey/import.sh

```
@@ -0,0 +1,2 @@
+# Ingestion keys are imported by name, not by ID
+terraform import groundcover_ingestionkey.example <name>
```

Shellcheck (0.11.0)

[error] Couldn't parse this redirection. Fix to allow more checks.

(SC1073)